exotica/doc/Installation.rst

```diff
@@ -23,6 +23,25 @@ Installation
 5. Compile the code ``catkin build -s``.
 6. Source the config file (ideally inside ``~/.bashrc``): ``source path_to_workspace/devel/setup.bash``. You may have to source the config file from your install-space if your workspace is configured for installation.
 
+CMakeLists.txt & package.xml
```

Comment:
I disagree with moving this section to Installation, it's part of getting started or usage.

Reply:
Yes, you're correct; I changed it over and altered the header title